### DISCUSSION TITLE
Componentes sin issue

### DISCUSSION THREAD

**Discussion by SantipBarber** [ArisGuimera/JetpackComposePro#49] [General]
Hola, estaba revisando el proyecto y las issues que hay creadas para colaborar en alguna y veo que ya están todas asignadas o alguien está solicitando que se le asignen. Pero, si voy al [catálogo](https://www.jetpackcompose.pro) veo que todavía quedan algunos componentes si documentar. Por ejemplo, topAppBar, Card, Bottom Navigation, Carousel... ¿Es porque no se han creado aún? ¿Las podemos crear nosotros mismos y asignárnoslas? A ver si podemos colaborar 😉

**Comment by ArisGuimera** [comment]
Claro, puedes crear issues siguiendo el patrón que utilizo y os las asigno yo!
